practice-app/templates/coronavirus.html

```diff
@@ -105,19 +105,13 @@
                 </div>
                 <nav class="navbar">
                   <ul class="navbar_menu">
-                    <li><a href="#">home</a></li>
+                    <li><a href="/">home</a></li>
```

Comment:
This href link is not working that way in flask. You can check other frontend pages for usage of {{ url_for() }} function.

Reply:
i checked  usage of {{ url_for() }} function, but my code seems fine too. It uses relative path linking, thank you for your review.

Reply:
Then it can be stay that way.